dev/projection/script-animate.js

```diff
@@ -33,7 +33,7 @@ Animate.createNode = (
     element,
     parent,
     options = {},
-    transition = { duration: 200, ease: () => 0.5 }
+    transition = { duration: 10, ease: () => 0.5 }
```

Comment:
Wouldn't this break some of our projection tests? I vaguely remember there are some tests with a 50ms timeout, but maybe I was wrong 😅

Reply:
That's ten seconds

Reply:
OH 🙈